src/Views/HardwareView.vala

```diff
@@ -162,17 +162,8 @@ public class About.HardwareView : Gtk.Grid {
     }
 
     private async void load_fallback_manufacturer_icon () {
-        try {
-            system_interface = yield Bus.get_proxy (
-                BusType.SYSTEM,
-                "org.freedesktop.hostname1",
-                "/org/freedesktop/hostname1"
-            );
-
-            manufacturer_logo.icon_name = system_interface.icon_name;
-        } catch (IOError e) {
-            critical (e.message);
-        }
+        get_system_interface_instance ();
+        manufacturer_logo.icon_name = system_interface.icon_name;
```

Comment:
We should probably still have a null check here in case `get_system_interface_instance ()` fails for some reason